tools/depends/native/TexturePacker/src/TexturePacker.cpp

```diff
@@ -286,7 +286,7 @@ void Usage()
   puts("  -help            Show this screen.");
   puts("  -input <dir>     Input directory. Default: current dir");
   puts("  -output <dir>    Output directory/filename. Default: Textures.xpr");
-  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: on");
+  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: off");
```

Comment:
nope. L458 sets dupecheck internaly if -dupecheck is used. default is already false at L426